addons/Cross_Lesson/src/presenter.js

```diff
@@ -260,6 +273,15 @@ function AddonCross_Lesson_create(){
             return;
         }
 
+        if (window.crossLessonAccessCache != null) {
+            if (window.crossLessonAccessCache[presenter.accessCacheId] !== undefined) {
+                if (!window.crossLessonAccessCache[presenter.accessCacheId]) {
+                    presenter.hide();
+                }
+                return;
```

Comment:
Zastanawiam się nad "return" gdyż w jego spełnienia, nie zostanie wysłany event poniżej

Reply:
Jest to celowe. Event ma być wysłany tylko w sytuacji, gdy nie otrzymaliśmy już odpowiedzi na niego wcześniej (czyli jeśli nie jest zapisany w crossLessonAccessCache).